src/components/editor/ModalToggle.jsx

```diff
@@ -0,0 +1,51 @@
+// Copyright 2018 Stanford University see Apache2.txt for license
+
+import React, {Component} from 'react'
+import PropTypes from 'prop-types'
+import ResourceTemplateModal from './ResourceTemplateModal'
+
+class ModalToggle extends Component {
+  constructor(props) {
+    super(props)
+    this.toggleModal = this.toggleModal.bind(this)
+    this.state = {
+      visible: false
+    }
+  }
+
+  toggleModal = () => {
+    const prevState = this.state
+    prevState.visible = !prevState.visible
+    this.setState(prevState => (prevState))
```

Comment:
i think you could also do this if you want: 

```
this.setState({
    visible: !this.state.visible
})
```

Reply:
I tried some stuff;   dunno if I tried this.